fix: typo in return type of `DateTime.time()`

mttbernardini opened the pull request.

mttbernardini committed:
fix: typo in return type of `DateTime.time()`

glyph commented:
Thank you @mttbernardini for your attention to detail!

glyph committed:
ignore dist

This commit was sponsored by Matt Campbell, Blaise Pabon, Cassie
Jones, and my other patrons.  If you want to join them, you can
support my work at https://glyph.im/patrons/.

glyph committed:
make this a bit less annoying to run in my terminal

This commit was sponsored by Blaise Pabon, Matt Campbell, and my other
patrons.  If you want to join them, you can support my work at
https://glyph.im/patrons/.

glyph committed:
expand tzinfo check to include explicit ignore

c.f. https://github.com/python/mypy/issues/4717

This commit was sponsored by Blaise Pabon, Matt Campbell, and my other
patrons.  If you want to join them, you can support my work at
https://glyph.im/patrons/.

glyph reviewed (approved):
Obviously the *fix* here is correct, so I will address the testing issues, which I think are incidental.

glyph committed:
drop EOL python versions to work in GHA CI

This commit was sponsored by Blaise Pabon, and my other patrons.  If
you want to join them, you can support my work at
https://glyph.im/patrons/.

glyph committed:
verify that we get an error

This commit was sponsored by Blaise Pabon, and my other patrons.  If
you want to join them, you can support my work at
https://glyph.im/patrons/.

glyph committed:
sigh, "best effort", I guess, given how core this library is

This commit was sponsored by Blaise Pabon, and my other patrons.  If
you want to join them, you can support my work at
https://glyph.im/patrons/.
